### PR TITLE
bugfix: s3c-2019 redis sentinel authentication

### PR DESCRIPTION
**Pull request by JianqinWang**
fix S3C-2019, include sentinel config support + password

### CODE REVIEW

**Comment by JianqinWang** [comment]
@bert-e reset

File: lib/Config.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ class Config extends EventEmitter {
                     item.substr(0, lastColon);
                 // the port should not include the colon
                 const port = item.substr(lastColon + 1);
-                assert(parseInt(port, 10),
+                assert(Number.parseInt(port, 10),
```
Comment (dora-korpar) [inline lib/Config.js:401]: should work with or without the `Number.`

Comment (JianqinWang) [reply lib/Config.js:401]: In 8.0/8.1 they're using `Number.parseInt`, I would prefer to keep this way to avoid additional merge conflicts :')

**Review by dora-korpar — APPROVED** [review]

**Comment by JianqinWang** [comment]
@bert-e reset

**Comment by JianqinWang** [comment]
@bert-e reset

**Review by rahulreddy — APPROVED** [review]

**Comment by JianqinWang** [comment]
@bert-e approve

**Pull request merged**